src/cli/subcommand/plan.rs

```diff
@@ -32,6 +32,8 @@ impl CommandExecute for Plan {
     async fn execute(self) -> eyre::Result<ExitCode> {
         let Self { planner, output } = self;
 
+        ensure_root()?;
```

Comment:
I wonder if this should be gated on being on a mac instead of always being necessary?

Reply:
I don't think it's fair to assume non-mac planners won't need to read files that an unpriviledged user cannot access. :(